socially/index.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+$request = $_SERVER['REQUEST_URI'];
+
+// Routes to the files.
+switch ($request) {
+  case '':
+  case '/':
+    session_start();
```

Comment:
Why do we need to start session in the index.php file ?

Reply:
Why can't we handle this logic in a controller ? What is the need for handling this in the index file itself ?

Reply:
as I'm using the same ' / ' path as login page and homepage, and this index.php file as router, I thought maybe this approach would work properly. Not sure about how to handle this in controller ..